docs/documentation/helpcenter.md

```diff
@@ -247,6 +248,12 @@ change `has_mac_keyboard` in `/static/js/common.ts` to always return `True`.
 Then when you view your documentation changes in the development environment,
 the keyboard shortcuts should be rendered with Mac keys where appropriate.
 
+If you're adding a tip to an article about a keyboard shortcut, you should use
+the more specific keyboard [tip macro](#tips-and-warnings), e.g.,
+`!!! keyboard_tip ""`. In general, all keyboard shortcuts should be documented
+on the [keyboard shortcuts](https://zulip.com/help/keyboard-shortcuts) help
```

Comment:
Hm, I was a bit confused by the "e.g." -- `keyboard_tip` is the only keyboard-specific macro, right? How about this?

> you should use
the more specific keyboard [tip macro](#tips-and-warnings) (
`!!! keyboard_tip ""`).